ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { VulnerabilitySeverity } from 'types/cve.proto';
+import { ImageType } from 'types/reportConfigurationService.proto';
+import { CVESDiscoveredSince, CVEStatus } from './forms/useReportFormValues';
+
+export const cveSeverityLabelMap: Record<VulnerabilitySeverity, string> = {
+    UNKNOWN_VULNERABILITY_SEVERITY: 'Unknown',
+    CRITICAL_VULNERABILITY_SEVERITY: 'Critical',
+    IMPORTANT_VULNERABILITY_SEVERITY: 'Important',
+    MODERATE_VULNERABILITY_SEVERITY: 'Moderate',
+    LOW_VULNERABILITY_SEVERITY: 'Low',
+};
+
+export const cveStatusLabelMap: Record<CVEStatus, string> = {
+    FIXABLE: 'Fixable',
+    NOT_FIXABLE: 'Not fixable',
+};
+
```

Comment:
Similarly with these, could you use https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/constants/reportConstants.ts instead?

Although severity and fixability are general enough that an argument can be made to put them in a file with a name less specific than `reportConstants`...